xmtp_mls/migrations/2024-11-05-191238_cache_wallet_addresses/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE wallet_addresses(
+    inbox_id TEXT NOT NULL,
```

Comment:
We'll probably want to create an index for this column alone, or just make it the primary key instead of using a compound primary key.

Reply:
Oh, I read the PR title and thought we're looking up addresses, but apparently it's the other way around? :smile: If so, then I mean the other column.